README.md

```diff
@@ -373,6 +374,13 @@ Returns a function that returns method result for a given object on predefined a
 $userIds = map(methodCaller('getId'), $users);
 ```
 
+##### instanceCreator($class)
+Returns a function that returns a new instance of a predefined class for given parameters
```

Comment:
How about `Returns a function that returns a new instance of a predefined class, passing its parameters to the constructor.`? I guess it's more precise.

Reply:
Sure, I will fix it